src/main/resources/zoo.cfg.template

```diff
@@ -0,0 +1,3 @@
+# clientPort:
+#     the port at which the clients will connect
+clientPort=##CLIENT_PORT##
```

Comment:
This file is only used by GigaSpaces and its not used for configuring the Zookeeper server instance itself. I have made this file minimal, containing only the `clientPort` used for finding the Zookeeper instance by GS.